style.css

```diff
@@ -196,35 +212,161 @@
   background-repeat: no-repeat;
   margin-left: 5px;
 }
-.thumbcontainer::after {
+.thumb__container::after {
   content: "";
   width: 108px;
   height: 1px;
   background-image: url(images/workdot-line.gif);
   background-repeat: no-repeat;
-  margin-top: 5px;
 }
 
-.topleft {
+.thumb__image {
+  width: 109px;
+  height: 105px;
+}
+.thumb__image.top__left {
   background-image: url(images/workimg1.jpg);
   background-repeat: no-repeat;
 }
-.bottomleft {
+.thumb__image.bottom__left {
   background-image: url(images/workimg2.jpg);
   background-repeat: no-repeat;
 }
-.topright {
+.thumb__image.top__right {
   background-image: url(images/workimg3.jpg);
   background-repeat: no-repeat;
 }
-.bottomright {
+.thumb__image.bottom__right {
   background-image: url(images/workimg4.jpg);
   background-repeat: no-repeat;
 }
 
+.quote__container{
+  display:flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height:150px;
+}
+.quote__container::after {
+  content: "";
+  height: 1px;
+  background-image: url(images/workdot-line.gif);
+}
+.quote__container p{
+  position: relative;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-size: 11pt;
+  font-style: italic;
+  color:#323232;
+  quotes: '\201c' '\201d';
+}
+
+.quote__container p::before {
+  position: absolute;
+  top:-15px;
+  left: -30px;
+  content: open-quote; 
+  color:#bf4d02;
+  font-size:40pt;
+  max-height:20px;
+  float:left;
+}
+.quote__container p::after {
+  content: close-quote; 
+  color:#bf4d02;
+  font-size:40pt;
+  max-height:20px;
+  float:right;
+}
+
+.quote__container span{
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-size: 14pt;
+  text-align: right;
+  font-weight: bold;
+  font-style: italic;
+  color:#ac8101
+}
+#developements__article__title__icon {
+  background-image: url(images/icon1.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+.support__image{
```

Comment:
Asta nu e afisata in pagina. Presupun ca acel forward slash de la inceput este de vina